src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.DevOps.tests.ps1

```diff
@@ -0,0 +1,37 @@
+Import-Module -Name $PSScriptRoot\..\Arcus.Scripting.DevOps -ErrorAction Stop
+
+InModuleScope Arcus.Scripting.DevOps {
+    Describe "Arcus Azure DevOps integration tests" {
+        BeforeEach {
+            $filePath = "$PSScriptRoot\appsettings.json"
+            [string]$appsettings = Get-Content $filePath
+            $config = ConvertFrom-Json $appsettings
+            
+            & $PSScriptRoot\Connect-AzAccountFromConfig.ps1 -config $config
+        }
+        Context "Save Azure DevOps build" {
+            It "Saves the Azure DevOps build indefinetely" {
+                # Arrange
+                $projectId = $env:SYSTEM_TEAMPROJECTID
+                $buildId = $env:BUILD_BUILDID
+                $collectionUri = $env:SYSTEM_COLLECTIONURI
+                $requestUri = "$collectionUri" + "$projectId/_apis/build/builds/" + $buildId + "?api-version=6.0"
```

Comment:
Does it make sense to rename this variable so that it is clear that this Uri is used to get the details of the build (to verify if the build is indeed retained ) ?
For instance `getBuildDetailsRequestUri` ?

Reply:
Well, in this case: no, because the same variable is used also later in the teardown-section where we revert the changes.